src/corehost/cli/deps_resolver.cpp

```diff
@@ -267,9 +243,18 @@ void deps_resolver_t::setup_additional_probes(const std::vector<pal::string_t>&
     }
 }
 
-bool deps_resolver_t::probe_entry_in_configs(const deps_entry_t& entry, pal::string_t* candidate)
+/**
+ * Given a deps entry, do a probe (lookup) for the file, based on the probe config.
+ *   -- When match hash is specified, the nuget cache SHA is matched. See .sha512 files in %USERPROFILE%\.nuget.
```

Comment:
Didn't we need hash matching for Antares cache as implemented today, which we no longer need post this change?